channel.js

```diff
@@ -0,0 +1,201 @@
+const SimpleMessageChannel = require('simple-message-channels')
+const { EventEmitter } = require('events')
+const duplexify = require('duplexify')
```

Comment:
Not used.

Reply:
```suggestion
```

Reply:
Also fwiw, Mathias mentioned he avoids duplexity these days.  I don't remember the reason.

Reply:
oof good to know! just removed it